cli/ioctl/cmd/account/account.go

```diff
@@ -9,24 +9,26 @@ package account
 import (
 	"bytes"
 	"context"
+	"encoding/hex"
 	"fmt"
 	"syscall"
 
 	"github.com/ethereum/go-ethereum/accounts/keystore"
-	"github.com/iotexproject/go-pkgs/crypto"
-	"github.com/iotexproject/iotex-address/address"
-	"github.com/iotexproject/iotex-proto/golang/iotexapi"
-	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"go.uber.org/zap"
 	"golang.org/x/crypto/ssh/terminal"
 	"google.golang.org/grpc/status"
 
+	"github.com/iotexproject/go-pkgs/crypto"
```

Comment:
change back to original import style, iotex-address, iotex-proto do not belong to iotex-core internal project